src/functions/tester/types.ts

```diff
@@ -0,0 +1,64 @@
+import type {
+  ParameterSetDefinition,
+  PossibleParameterKeys,
+} from "../../parameters/mod.ts";
+import type { SlackFunction } from "../mod.ts";
+import type { FunctionContext, FunctionRuntimeParameters } from "../types.ts";
+
+export type SlackFunctionTesterArgs<
+  InputParameters,
+> =
+  & Partial<
+    FunctionContext<InputParameters>
+  >
+  & {
+    inputs: InputParameters;
+  };
+
+export type CreateContext<
+  InputParameters extends ParameterSetDefinition,
+  RequiredInput extends PossibleParameterKeys<InputParameters>,
+> = {
+  (
+    args: SlackFunctionTesterArgs<
+      FunctionRuntimeParameters<InputParameters, RequiredInput> | undefined
+    >,
+  ): FunctionContext<
+    FunctionRuntimeParameters<InputParameters, RequiredInput>
+  >;
+};
```

Comment:
This `CreateContext` type is new, but unfortunately couldn't figure out how to re-use it